app/controllers/admin/student_accounts_controller.rb

```diff
@@ -0,0 +1,37 @@
+require 'date'
+
+module Admin
+    class StudentAccountsController < BaseController
+      layout 'admin'
+  
+      def actions
```

Comment:
Since [some newer version of Rail, I think Rails 5], you don't need to define a function in the controller for GET requests. So unless you're defining an `@instance_variable` or something like that, I'd suggest removing this function to make the controller cleaner ™️ (creating clean controllers, and models, is a big problem in Rails).

Reply:
The page will not display without actions being defined. I originally was using show, but Dante had me change it to actions.